### PR TITLE
Tues sprint1 task4 amb

### LINKED ISSUE #21: [Task] Task 3:  Implement Azure Function for /user{user_id}/tasks
##  Build '/user{user_id}/tasks' API endpoint using Azure Functions
- [x] Read/ Plan/ Prototype
```
- Required Methods and URL path
- Database query handler
- Http Request handler
```
- [x] Build
- [x] Test

### LINKED ISSUE #26: Task 4: Implement Azure Function for /users/{user_id}/tasks/{task_id}
Andrew Boyer

**Is your feature request related to a problem? Please describe.**
The project requires Azure Function to call the /users/{user_id}/tasks/{task_id} API

**Describe the solution you'd like**
A functional API for /users/{user_id}/tasks/{task_id}

**Describe alternatives you've considered**
N/A

### PR DESCRIPTION
**Pull request by Raotrux**
I've been able to connect to the database and successfully run the GetTasks function. However logic for other functions has yet to be implemented as getting the development environment set up correctly on my PC was difficult and took up the majority of our time.

### CODE REVIEW

**Comment by All-Schuck-Up** [comment]
Is It possible or advisable to add the .venv files to the gitignore?  As I was reviewing I noticed that there are 500+ pip package files.

**Comment by Raotrux** [comment]
I added '.venv/' and '.vscode/' to the gitignore but it still committed some of them anyways. I'm sure I did something wrong but I'll look into it and fix that.

Edit:
I added these to my gitignore at the same time the .venv files were being commited. Since they were commited and tracked by github, they weren't removed. I moved my .venv and .vscode files out of the folder, commited the change (it shows you remove the applicable files), and then moved them back in.

**Comment by Raotrux** [comment]
Alright, all of my .vscode and .venv files are added to my gitignore and seem to be gone from PR now.

**Review by RealDogDad — APPROVED** [review]
Specifying the trigger type?

**Comment by Raotrux** [comment]
> Specifying the trigger type?

What's this in reference to?

Also, I've committed code to my Task 4 functions. GetTask, DeleteTask, PutTask are all functioning for me. So I'd recommend someone else review and approve as well.

File: .gitignore [hunk 1]
```diff
@@ -102,3 +102,12 @@ dist
...
 .tern-port
+
+# VS Code Workspace config files
+*.code-workspace
+
+# Mac window config files
+.DS_Store
+
+.venv/
+.vscode/
```
Comment (jlsnow301) [inline .gitignore]: Oh nice. This closes #49

**Review by jlsnow301 — APPROVED** [review]

**Review by All-Schuck-Up — APPROVED** [review]

**Comment by Raotrux** [comment]
This is ready to merge. I don't know if I'm supposed to do it since I created this PR or if that's the reason I shouldn't be the one to do it. I guess someone let me know if I need to do it.

**Pull request merged**